app/controllers/v0/sessions_controller.rb

```diff
@@ -37,7 +37,13 @@ def saml_callback
         obscure_token = Session.obscure_token(@session.token)
         Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
       else
-        log_errors
+        message = <<-MESSAGE.strip_heredoc
```

Comment:
Before this was inside `log_errors`, and was brought out so that method could be repurposed. Could it still be a separate method that calls `log_errors`?